src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,330 @@
+package kitchenpos.application;
+
+import jakarta.transaction.Transactional;
+import kitchenpos.config.MenuTestContextConfiguration;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import kitchenpos.helper.MenuGroupTestHelper;
+import kitchenpos.helper.MenuProductTestHelper;
+import kitchenpos.helper.MenuTestHelper;
+import kitchenpos.helper.ProductTestHelper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Import;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+@Transactional
+@Import(MenuTestContextConfiguration.class)
+class MenuServiceTest extends SetupTest{
+    @Autowired
+    private MenuService menuService;
+
+    private MenuGroup 추천메뉴;
+    private MenuProduct 마라탕메뉴;
+    private MenuProduct 미니꿔바로우메뉴;
+    private MenuProduct 콜라메뉴;
+    private Product 마라탕;
+    private Product 미니꿔바로우;
+    private Product 콜라;
+    private Menu 마라세트;
+    private List<Menu> 메뉴들 = new ArrayList<>();
+
+
+    @BeforeEach
+    void setUp() {
+        super.setUp();
+
+        추천메뉴 = MenuGroupTestHelper.메뉴카테고리_생성("추천메뉴");
+
+        마라탕 = ProductTestHelper.음식_생성("마라탕", BigDecimal.valueOf(10000));
+        미니꿔바로우 = ProductTestHelper.음식_생성("미니꿔바로우", BigDecimal.valueOf(8000));
+        콜라 = ProductTestHelper.음식_생성("콜라", BigDecimal.valueOf(3000));
+
+        마라탕메뉴 = MenuProductTestHelper.음식메뉴_생성(마라탕, 1);
+        미니꿔바로우메뉴 = MenuProductTestHelper.음식메뉴_생성(미니꿔바로우, 1);
+        콜라메뉴 = MenuProductTestHelper.음식메뉴_생성(콜라, 1);
+
+        마라세트 = MenuTestHelper.메뉴_생성(추천메뉴, "마라세트", BigDecimal.valueOf(16000), Arrays.asList(마라탕메뉴, 미니꿔바로우메뉴), true);
+
+        메뉴들.add(마라세트);
+    }
+
+    @DisplayName("특정 메뉴카테고리에 음식메뉴를 조합한 신규 메뉴를 추가한다.")
```

Comment:
테스트 이름을 정리해주신 요구사항과 통일시켜보는건 어떻게 생각하시나요?

```suggestion
    @DisplayName("매장주인은 특정 메뉴 카테고리에 음식을 조합해서 메뉴를 등록할 수 있다.")
```

Reply:
추가 대신 등록으로 표현하는게 맞겠네용 
수정했습니다!